src/main/java/net/neoforged/nfrtgradle/DownloadedAssetReference.java

```diff
@@ -0,0 +1,29 @@
+package net.neoforged.nfrtgradle;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Properties;
+
+public record DownloadedAssetReference(String assetIndex, String assetsRoot) {
```

Comment:
Call it `DownloadedAssetsReference` maybe. Singular refers to a single asset. Or maybe simply `AssetProperties`. Note that this record is now being made public API.

Reply:
(I just think `DownloadedAssetReference` is a bad name.)